plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -948,14 +951,10 @@ public Optional<ConnectorTableExecuteHandle> getTableHandleForExecute(
         }
 
         switch (procedureId) {
```

Comment:
use `return` and drop the subsequent IAE - the compiler will do in this case the exhaustiveness check in the switch.

Reply:
ok. done.